students/JonathanMauk/lesson04/trigram.py

```diff
@@ -0,0 +1,66 @@
+import random
+import re
+import textwrap
+
+
+def get_file_text(filename):
+    with open(filename, 'r') as f:
+        text = f.read()
+        f.close()
```

Comment:
The `with` statement will automatically close the file for you when you exit the block